action.yml

```
@@ -5,6 +5,8 @@ branding:
   color: green
 
 inputs:
+  REMOTE_NAME:
+    description: "Remote name"
   REMOTE:
```

```suggestion
    description: "Remote name"
    required: false
  REMOTE:
```

I forget if this is needed or not.